dummyserver/server.py

```diff
@@ -40,6 +40,20 @@
 DEFAULT_CA_KEY = os.path.join(CERTS_PATH, "cacert.key")
 
 
+def _sane_ipv6_dns():
```

Comment:
Can we please avoid using `sane` here? I know we're using the word "sanity" elsewhere in the project, but I'd like to avoid using it more. `_resolves_to_ipv6` avoids this issue and is also more descriptive.

Reply:
Done :)